cmd/ocsp-updater/main.go

```diff
@@ -129,11 +140,63 @@ func newUpdater(
 		})
 	}
 
-	updater.ocspMinTimeToExpiry = config.OCSPMinTimeToExpiry.Duration
+	if config.AkamaiBaseURL != "" {
```

Comment:
I think it might actually make sense to leave this gated indefinitely. LE uses Akamai as a CDN provider but future users may not...?